tests/pyproject.toml

```diff
@@ -33,6 +33,5 @@ pyodide.test-groups = ["numpy", "scipy"]
 ios.test-groups = ["numpy"]
 ios.xbuild-tools = ["cmake", "ninja"]
 ios.environment.PIP_EXTRA_INDEX_URL = "https://pypi.anaconda.org/beeware/simple"
-android.test-groups = ["numpy"]
-android.xbuild-tools = ["cmake", "ninja"]
```

Comment:
> android.test-groups = ["numpy"]

Unfortunately the Chaquopy NumPy wheel won't work with the CPython testbed, as it packages OpenBLAS in a Chaquopy-specific location.

> `android.xbuild-tools = ["cmake", "ninja"]`

xbuild-tools is not implemented on Android: https://github.com/pypa/cibuildwheel/pull/2349#issuecomment-2944926737